.github/workflows/publish.yml

```
@@ -0,0 +1,53 @@
+name: Release Pipeline (develop/2.0)
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - labeled
+    check_suite:
+      types:
+      - completed
+
+jobs:
+  release-to-github:
+    needs:
+    - merge-release-branch
+    # release to github only when its a prerelease branch and auto merge label is triggered
+    if: ${{ startsWith(github.head_ref, 'useOnlyCENNZnetExtension') }} && ${{ github.event.label.name == 'automerge' }}
+    runs-on: ubuntu-latest
+    steps:
+    # checkout merged branch
+    - uses: actions/checkout@v2
+      with:
+        ref: useOnlyCENNZnetExtension
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '15.3'
+    - name: find release version
+      id: find_version
+      run: |
+        echo ::set-output name=tag_name::$(cat packages/extension/package.json \
+        | grep version \
+        | head -1 \
+        | grep -E -o '[0-9|.]+\-*[a-z]*\.[0-9]+|]+')
+    - name: make release branch
+      uses: peterjgrainger/action-create-branch@v2.0.1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        branch: "release/${{steps.find_version.outputs.tag_name}}"
+    - name: package
+      run: |
+        yarn && yarn build
+        npx lerna exec -- 'cp package.json build/'
+        npx lerna exec -- 'cp README.md build/'
+        npx lerna exec -- 'cp ../../LICENSE build/'
+    - name: publish to npm
+      env:
+        NPM_KEY: ${{ secrets.NPM_KEY }}
+      run: |
+        echo "//registry.npmjs.org/:_authToken=${NPM_KEY}" > ~/.npmrc
+        echo 'Publishing to npmjs.org.'
+        npx lerna exec -- 'cd build && npm publish --access public'
```

can add `--dry-run` if you want to test the CI before publishing
```suggestion
        npx lerna exec -- 'cd build && npm publish --access public'
```